### ISSUE TITLE
Evaluation metrics: mAP

### ISSUE THREAD

**Issue by off99555** [thtrieu/darkflow#588]
How do I evaluate mAP (mean average precision) of my own dataset?
@thtrieu on issue #70 suggested outputting the JSON file and then create a script to report mAP on your own.
But anyone did that already?
If so please share the script.

I need a way to evaluate my test set or my training set. If mAP is not possible then loss is acceptable too.

**Comment by mdawas** [comment]
check this, https://github.com/mdawas/mAP
P.S: It's forked from the original repo (i.e. not my own work)

**Comment by off99555** [comment]
It's been long and I used this repo at that time: https://github.com/Cartucho/mAP
It also works fine.
